.rubocop.yml

```diff
@@ -12,10 +12,12 @@ AllCops:
   TargetRubyVersion: 2.6
 
   Exclude:
-    - 'db/schema.rb'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'bin/yarn'
```

Comment:
I don't like to ignore all of these. Any reason in particular? :)

Reply:
I don't usually use code analysis tools in config and auto generated files. I'd be happy to include those files if you think it is worth the effort to analyze them.

Reply:
I'm happy to ignore the auto generated files, but DB migrations and `config` folder might have lots of custom code. Does it make sense? ;)

Reply:
By excluding `db/schema.rb`, most of offenses should go away :)

Reply:
@lucascaton I have another issue when removing `db` folder from the config file. I think those cops aren't useful because database folder and `spec` have a particular dsl. So I created the following config [here](8ca696e) and [here](d73ec9b)

```
Inspecting 35 files
...C........C......................

Offenses:

spec/decorators/live_decorator_spec.rb:3:1: C: Metrics/BlockLength: Block has too many lines. [27/25]
RSpec.describe LiveDecorator do ...
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
db/migrate/20190406000855_devise_create_users.rb:4:3: C: Metrics/MethodLength: Method has too many lines. [16/10]
  def change ...
  ^^^^^^^^^^
```

Reply:
Awesome, thanks @thiagoalessio :)

Reply:
Awesome, thanks @tmmgarcia! :)